src/collective/behavior/amp/browser/view.pt

```diff
@@ -54,6 +54,23 @@
     <p tal:condition="context/text/output|nothing"
         tal:replace="structure context/text/output">Body text</p>
 
+    <div class="amp-related">
+      <span i18n:translate="label_related_items">Related content</span>
+      <ul tal:define="site_properties context/portal_properties/site_properties;
```

Comment:
@rodfersou the portal_properties tool has been removed in Plone 5, so this is not going to work; please find out how to make this compatible with both versions.